src/po/pl.po

```diff
@@ -410,81 +425,81 @@ msgid "Maximize"
 msgstr "Maksymalizuj"
 
 msgid "Unmaximize"
-msgstr ""
+msgstr "Zmniejszenie"
 
 msgid "Restore"
-msgstr "Przywróc"
+msgstr "Przywróć"
 
 msgid "Use theme padding to center button icons"
-msgstr ""
+msgstr "Użyj ustawień przestrzeni motywów do wyśrodkowania ikon przycisków"
 
 msgid "If button icons look off-centered, toggling this may help."
-msgstr ""
+msgstr "Jeśli wygląd przycisków ikon jest nie do końca wyśrodkowany, przełączenie tego spróbuje pomóc."
 
 msgid "App button hover pseudo class"
-msgstr ""
+msgstr "Pseudo klasa dla przycisku aplikacji po najechaniu"
 
 msgid "Overrides the pseudo class for app buttons on hover."
-msgstr ""
+msgstr "Nadpisuje pseudo klasę dla przycisków aplikacji po najechaniu na nią."
 
 msgid "App button focus pseudo class"
-msgstr ""
+msgstr "Pseudo klasa dla przycisku aktywnego okna aplikacji"
 
 msgid "Overrides the pseudo class for app button on focus."
-msgstr ""
+msgstr "Nadpisuje pseudo klasę dla przycisku aplikacji przy wybraniu aktywnego okna."
 
 msgid "App button active pseudo class"
-msgstr ""
+msgstr "Pseudo klasa dla przycisku aplikacji gdy aktywny"
 
 msgid "Overrides the pseudo class for app buttons when they are active."
-msgstr ""
+msgstr "Nadpisuje pseudo klasę dla przycisków aplikacji kiedy są aktywne."
 
 msgid "Use the panel launcher class for pinned apps"
-msgstr ""
+msgstr "Używaj klasy panelu uruchamiania dla przypiętych aplikacji"
 
 msgid "Enabling this helps with certain themes."
-msgstr ""
+msgstr "Włączenie tego pomaga przy różnych motywach."
 
 msgid "Global hotkey for cycling through thumbnail menus"
-msgstr ""
+msgstr "Globalny skrót klawiatury do przechodzenia pomiędzy miniaturkami menu"
 
 msgid "This allows you to cycle through the menus without activating a window."
-msgstr ""
+msgstr "To pozwoli ci na przechodzenie po menu bez aktywowania okna."
 
 msgid "Update this applet automatically"
-msgstr ""
+msgstr "Aktualizuj ten aplet automatycznie"
 
 msgid ""
-"When enabled, ITM will check the Github releases on init, and will "
-"start to upgrade if it finds a newer version."
+"When enabled, ITM will check the Github releases on init, and will start to upgrade if it finds a newer "
```

Comment:
Hmm... that's strange. Po editor have already set up option to "wrap" text after 80 characters...
Anyway, I'll fix that.
Translated strings also shouldn't exceed 80 characters?